src/database/config/mongoose-config.ts

```diff
@@ -0,0 +1,6 @@
+import mongoose from "mongoose";
+
+const uri = `mongodb+srv://${process.env.MONGO_HOST}:${process.env.MONGO_PASSWORD}@cluster0.dkvjg.mongodb.net/todo?retryWrites=true&w=majority`;
+mongoose.connect(uri).catch((err) => {
+  console.log(err, "err");
```

Comment:
since the whole app relies on the database connection, it would be better to throw some exception instead of just logging error